app/api/revalidate/route.ts

```diff
@@ -49,6 +50,15 @@ const indexNow = async (uri: string) => {
   console.info(`${status}: ${indexNowStatusText[status]}`);
 };
 
+const revalidateChildren = (parts: Array<string>): "layout" | "page" => {
```

Comment:
since the assets in galleries are not represented in Craft, they need a new revalidation pipeline. Next has two types of revalidation: `page` which revalidates only the specified URL and `layout` which revalidates the URL + it's children.

In this exception, we want to use `layout` so that when the gallery entry is saved (either manually or by webhook), the assets are also revalidated to pick up any changes